### PR TITLE
Update wrap and farm detail layout components

### PR DESCRIPTION
**Pull request by bitsforpeace**
### What does this PR do?

Adapts Wrap and Farm page detail layouts to use new components

### CODE REVIEW

**Review by codewiththeflowxyz — APPROVED** [review]
LGTM!

As a note, I think we should start adding translations on the areas we work. No need to be done here. wdyt?

[LGTM](https://lgtm.lol/i/77)

**Comment by inclinedmind** [comment]
Screen Shot 2023-07-26 at 11 56 16 AM

Screen Shot 2023-07-26 at 11 56 08 AM

Noticing this on the deployment (for some reason connecting the wallet causes the farm text to show up)

File: apps/webapp/src/modules/farms/components/FarmDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </DetailSectionRow>
+        <DetailSection title="Metrics">
+          <DetailSectionRow>
+            <FarmCharts farm={farm} />
+          </DetailSectionRow>
+        </DetailSection>
+      </DetailSection>
+    </DetailSectionWrapper>
+  );
+}
```
Comment (inclinedmind) [inline apps/webapp/src/modules/farms/components/FarmDetails.tsx:36]: nice and clean :)

**Review by inclinedmind — APPROVED** [review]
LGTM [LGTM](https://lgtm.lol/i/811)

just pushed a commit to update .po files to fix nav text issue

**Pull request merged**